test/unit/components/date.tsx

```diff
@@ -73,12 +68,17 @@ describe('<FormattedDate>', () => {
     expect(rendered.text()).toBe(intl.formatDate(date, options));
   });
 
-  it('fallsback and warns on invalid Intl.DateTimeFormat options', () => {
+  it('falls back and warns on invalid Intl.DateTimeFormat options', () => {
     const date = new Date();
     const rendered = mountWithProvider({value: date, year: 'invalid'}, intl);
 
     expect(rendered.text()).toBe(String(date));
-    expect(consoleError.mock.calls.length).toBeGreaterThan(0);
+    expect(console.error).toHaveBeenCalledTimes(1);
+    expect(console.error).toHaveBeenCalledWith(
+      expect.stringContaining(
+        'Error formatting date.\nRangeError: Value invalid out of range for dateformat options property year'
```

Comment:
I added more expectations to check for the error message, as it gives a better understanding of the expected behavior.